src/caustics/__init__.py

```diff
@@ -1,23 +1,11 @@
 from ._version import version as VERSION  # noqa
 
-from . import constants, lenses, cosmology, packed, parametrized, light, utils, sims
-from .tests import test
-
-# from .demo import *
+from .cosmology import *
```

Comment:
To match with proper import formatting, every object must be imported explicitly. So this should become:
`from .cosmology import Cosmology, FlatLambdaCDM, h0_default, critical_density_0_default, Om0_default`
The other imports should also write out everything explicitly. It's a pain, but it only needs to be done once and then it's a nice format.